src/libraries/System.Threading.Channels/ref/System.Threading.Channels.csproj

```diff
@@ -1,6 +1,6 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>$(NetCoreAppCurrent);netcoreapp3.0;netstandard1.3;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>$(NetCoreAppCurrent);netcoreapp3.0;netstandard1.3;netstandard2.0;netstandard2.1</TargetFrameworks>
```

Comment:
@joperezr, is anything else required around packaging to correctly advertize/package the new netstandard2.1 assets?

Reply:
no, package should contain the new netstandard2.1 asset after these changes. This package in particular doesn't include the refs int he packages, but the libs act as refs and we should now have a lib/netstandard2.1 asset.

Reply:
Thanks.